gems/pending/spec/miq_vm/miq_azure_vm_image_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative './miq_vm_instance_methods_shared_examples'
+require 'ostruct'
+require 'azure-armrest'
+require 'MiqVm/miq_azure_vm'
+require 'vcr'
```

Comment:
It wasn't in the gems/pending/spec/spec_helper.rb initially. I only added it to the spec_helper when I added the VCR configuration. It's also probably in test_env_helper.rb, but that makes it easier to test in isolation of the Rspec environment. I'm inclined to keep them in, it doesn't really hurt and it makes the dependency obvious.